checker/src/main/resources/javadoc.astub

```diff
@@ -1,127 +0,0 @@
-// This file, javadoc.astub, models the JavaDoc classes in the JDK 8's
```

Comment:
There should be a pull request into typetools/jdk where these annotations are moved, right?

Reply:
All the annotations are already in typetools/jdk.

Reply:
So this branch, https://github.com/mernst/jdk/tree/remove-javadoc-astub, can be deleted?